### PR TITLE
Adds the Coderbus bar sign

### PR DESCRIPTION
**Pull request by ghost**
thecoderbus

I saw this suggestion in the PR for re-adding the Grey Tide sign and felt I needed to do it justice.
Also changes the description on all the bar signs (usually just adding periods).

I didn't see the need to add a changelog for something so minor but if you want one lemme know.

### CODE REVIEW

File: code/game/objects/structures/barsigns.dm [hunk 1]
```diff
@@ -64,52 +64,52 @@
 /datum/barsign/maltesefalcon
 	name = "Maltese Falcon"
 	icon = "maltesefalcon"
-	desc = "The Maltese Faclon, Space Bar and Grill"
+	desc = "The Maltese Faclon, Space Bar and Grill."
```
Comment (MrSnapwalk) [inline code/game/objects/structures/barsigns.dm]: Falcon, not Faclon.

**Comment by Mandurrrh** [comment]
Heck yes!

File: code/game/objects/structures/barsigns.dm (hunk 1 above)
Comment (Xhuis) [inline code/game/objects/structures/barsigns.dm]: Oh hey, that was in the original PR. I'll fix that.

**Comment by phil235** [comment]
:+1:

**Comment by RemieRichards** [comment]
Why not :+1:

**Comment by Fox-McCloud** [comment]
awesome.

**Comment by dannno** [comment]
i saw the PR name and i thought it was pretty gay but the sprite is cute so why not

**Comment by xxalpha** [comment]
I think the letters could be less plain but good nonetheless. :+1:

**Comment by paprka** [comment]
>not using the github font

ya blew it

**Comment by Xhuis** [comment]
:fu:  :monkey_face:

**Comment by Razharas** [comment]
What about copyright?
You sure none of headcoders will be vanned for this?
Or github deleteing our branch in retaliation?

**Comment by Xhuis** [comment]
:cop: :e: :arrow_right: police open up
:man: :gun: :cop:

**Comment by hornygranny** [comment]
:cop:	📧➡

**Comment by Xhuis** [comment]
Tbh if copyright is a legit issue I could make it the Coderbus or something very easily.

**Comment by GunHog** [comment]
Love it :+1: 10/10 nerdy!

**Comment by Cheridan** [comment]
@Razharas I dunno what the status of the octocat symbol is, but I assume it's protected in some way. 

@Xhuis I do like the sound of that. Maybe even just "The 'Bus", to encompass adminbus and coderbus at once?

**Comment by hornygranny** [comment]
"The Short Bus"

**Comment by optimumtact** [comment]
https://github.com/logos I'm not sure if we meet their criteria for using, but I suggest they'd be happy to clarify if we email them and ask.

**Comment by Cheridan** [comment]
Ah yeah, the terms there specifically disallows creating a modified version, or changing the dimensions or adding text. I don't think this would be allowed, then. Thanks for digging that page up.

**Comment by pudl** [comment]
the sprite itself is pretty bad, the idea isnt

**Comment by Niknakflak** [comment]
More bar signs!  Catchy names or puns are always great, and color variances.

**Comment by dannno** [comment]
^ this is just one of those things where we can go crazy with it and its great
the idea is great, so no matter how much subtlety you have to mire this in its gotta happen

**Comment by Xhuis** [comment]
All right, I will change this when I get home.

**Comment by Xhuis** [comment]
Sprite changed. OP is updated with the new .gif.

File: code/game/objects/structures/barsigns.dm [hunk 2]
```diff
@@ -120,3 +120,8 @@
 	name = "Honked 'n' Loaded"
 	icon = "honkednloaded"
 	desc = "Honk."
+
+/datum/barsign/coderbus
+	name = "The Coderbus"
+	icon = "thecoderbus"
+	desc = "The Coderbus is a very controversial establishment known for its large variety of drinks."
```
Comment (GunHog) [inline code/game/objects/structures/barsigns.dm]: How about "known for constantly changing its drinks." ? Seems more appropriate for how often the code is changed!

File: code/game/objects/structures/barsigns.dm [hunk 3]
```diff
@@ -64,52 +64,52 @@
...
 	name = "The Cavern"
 	icon = "thecavern"
-	desc = "Fine drinks while listening to some fine tunes"
+	desc = "Fine drinks while listening to some fine tunes."
 
 /datum/barsign/theouterspess
 	name = "The Outer Spess"
 	icon = "theouterspess"
-	desc = "This bar isn't actually located in outer space"
+	desc = "This bar isn't <i>actually</i> located in outer space."
```
Comment (RemieRichards) [inline code/game/objects/structures/barsigns.dm]: but... but it is... it's on a space station... I NEED AN ADULT.

Comment (Xhuis) [inline code/game/objects/structures/barsigns.dm]: Hey man, I didn't write the descriptions :alien:

Comment (RemieRichards) [inline code/game/objects/structures/barsigns.dm]: But you did touch them! Making that line valid for change in this PR :3

**Comment by Xhuis** [comment]
Incorporated some changes to descriptions.

**Comment by pudl** [comment]
the biggest issue is that the text simply isn't big enough, so it's very empty.

**Comment by paprka** [comment]
This does not appease my grey tide ptsd triggering over the bar sign incident

It's kind of cute but not very clever

**Comment by Nienhaus** [comment]
I feel like the sprite could be a bit better.

**Comment by Xhuis** [comment]
So do I. It was a codersprite.

**Comment by optimumtact** [comment]
I assumed that was part of the joke

**Comment by dannno** [comment]
Looks good to me, tbh. 
In my opinion, you should change it to "The Bus" and add in some of that noise effect that the gif you took achieves (I'm assuming it's all flat color ingame)

**Comment by Miauw62** [comment]
I think that "the back of the bus" is probably a better name.
Op 27-feb.-2015 01:22 schreef "Xhuis" <notifications@github.com>:

**Comment by Xhuis** [comment]
I'm gonna make this on a new branch - it's very finnicky

**Pull request closed without merging**